scripts/Engineering/EnginX.py

```diff
@@ -111,57 +128,86 @@ def run(ceria_run, do_cal, do_van, van_run, calibration_directory, calibration_g
 
     # if the calibration files that this run would create are not present, or the user has requested it, create the
     # calibration files
-    if not all(expected_cals_present) or do_cal:
-        create_calibration(ceria_run, van_run, calibration_directory, calibration_general, cropped, crop_name, crop_on)
+    if not all(expected_cals_present) or not all(expected_tables_present) or do_cal:
+        create_calibration(ceria_run, van_run, full_inst_calib, calibration_directory, calibration_general, cropped,
+                           crop_name, crop_on)
+    else:
+        ending_to_load = {"banks": f"bank_{crop_on}",
+                          "spectra": crop_name,
+                          None: "all_banks"}
+        file_name = os.path.join(calibration_directory,
+                                 f"ENGINX_{van_run}_{ceria_run}_{ending_to_load.get(cropped)}.prm")
+        Utils.load_relevant_pdcal_outputs(file_name, "engg")
 
     # if a focus is requested, run the focus
     if focus_run is not None:
-        focus(focus_run, van_run, calibration_directory, focus_directory, focus_general, do_pre_process, params,
-              time_period, grouping_file, cropped, crop_on)
+        focus(focus_run, van_run, full_inst_calib, calibration_directory, focus_directory, focus_general,
+              do_pre_process, params, time_period, grouping_file, cropped, crop_on)
 
 
-def create_vanadium(van_run, calibration_directory):
+def create_vanadium_integration(van_run, calibration_directory):
     """
-    create the vanadium run for the run number set of the object
+    create the vanadium integration for the run number set of the object
 
     @param van_run :: the run number for the vanadium
     @param calibration_directory :: the directory to save the output to
     """
     # find and load the vanadium
     van_file = _gen_filename(van_run)
-    van_curves_file, van_int_file = _get_van_names(van_run, calibration_directory)
+    _, van_int_file = _get_van_names(van_run, calibration_directory)
     van_name = "eng_vanadium_ws"
-    simple.Load(van_file, OutputWorkspace=van_name)
-
-    # make the vanadium corrections
-    simple.EnggVanadiumCorrections(VanadiumWorkspace=van_name,
-                                   OutIntegrationWorkspace="eng_vanadium_integration",
-                                   OutCurvesWorkspace="eng_vanadium_curves")
+    van_ws = simple.Load(van_file, OutputWorkspace=van_name)
 
+    # make the integration workspace
+    simple.NormaliseByCurrent(InputWorkspace=van_ws, OutputWorkspace=van_ws)
+    # sensitivity correction for van
+    ws_van_int = simple.Integration(InputWorkspace=van_ws)
+    ws_van_int /= van_ws.blocksize()
     # save out the vanadium and delete the original workspace
-    simple.SaveNexus("eng_vanadium_integration", van_int_file)
-    simple.SaveNexus("eng_vanadium_curves", van_curves_file)
+    simple.SaveNexus(ws_van_int, van_int_file)
     simple.DeleteWorkspace(van_name)
```

Comment:
Very minor, but the algorithm `DeleteWorkspaces` can delete a list of workspaces - I've seen repeated calls to `DeleteWorkspace` in a  few places but it's not a big deal.